data/hydraulic-data.py

```diff
@@ -0,0 +1,129 @@
+"""Create hydraulic systems dataset"""
+
+from typing import Union, List
+from h2oaicore.data import CustomData
```

Comment:
style: prefer to import modules and not the module level objects.

From this very helpful review comment: https://github.com/h2oai/h2oai-platform/pull/65#discussion_r412967978

Reply:
Thank you for providing these stylistic practices feedback! I can see how it helps in code readability improvements. I will make sure to make this a habit.